tests/ci_visibility/api/test_internal_test_visibility_api.py

```diff
@@ -13,6 +15,22 @@ class TestCIITRMixin:
     Note: these tests do not bother discovering a session as the ITR functionality currently does not rely on sessions.
     """
 
+    @pytest.fixture(scope="function", autouse=True)
+    def _disable_ci_visibility(self):
+        try:
+            if CIVisibility.enabled:
+                CIVisibility.disable()
```

Comment:
What do you think about making `CIVisibility.disable()` idempotent so we don't have to always check if it's enabled first?

Reply:
I think it's a great idea for another PR...